src/spaces/ictcp.js

```diff
@@ -26,6 +26,7 @@ Color.defineSpace({
 		CT: [-0.5, 0.5],	// Full BT.2020 gamut in range [-0.5, 0.5]
 		CP: [-0.5, 0.5]
 	},
+	classification: ['labish'],
```

Comment:
I like the idea of classifying color spaces, but it needs broader discussion about the API to do so.

What possible classifications do we envision? Are they mutually exclusive? Which of those can be inferred and which need to be explicitly specified? etc 

And what are the requirements for `labish`? E.g. `oklab` and `jzazbz` have completely different ranges than `lab`.

Reply:
My current requirements for labish is... basically the lightness-chroma_a-chroma_b layout. Some other classifications I am thinking of are:
* Element order: `rgb`, `lchish`, `hslish`
* Maybe some other properties: `perceptual`

It's made into a list for the possibility of having many tags.

Reply:
@svgeesus thoughts?

Reply:
So, OKLab is certainly `labish` with L, a and b axes. But the ranges are different, [0,1] for OKLab L for example. And certainly, one would not want to apply something like deltaE2000 to OKLab since the various corrections in deltaE 2000  (blue hue non-uniformity, notably) do not apply to OKLab.

Reply:
Only some deltaE methods will be able to take a space parameter, so I guess that is all fine and dandy.